### DISCUSSION TITLE
LG Companion for Linux?

### DISCUSSION THREAD

**Discussion by Faceless3882** [JPersson77/LGTVCompanion#193] [Ideas]
Hi @JPersson77 

I really love LG companion in windows, it's so useful and "just works". Any chance of looking into a linux version? Will be happy to donate if its at all possible?

Thanks!

**Comment by JPersson77** [comment]
Hi @Faceless3882 

Thanks! There has been some discussion around porting but nothing tangible. There is this discussion: https://github.com/JPersson77/LGTVCompanion/issues/81

If you are willing to invest a little time you can probably set up some scripts with [bscpylgtv](https://github.com/chros73/bscpylgtv) to achieve some functoinality

**Reply by Faceless3882** [reply]
I did a thing: https://github.com/Faceless3882/LG_Buddy

**Comment by JPersson77** [comment]
Nice well done!

**Comment by alimbada** [comment]
I've been using https://github.com/klattimer/LGWebOSRemote on macOS. It should work on Linux as it's built with Python. I use it along with a Hammerspoon script (https://github.com/cmer/lg-tv-control-macos) which automates the TV on/off functionality on power events. You should be able to do something similar on Linux.

**Comment by xpusostomos** [comment]
Simple tool here that can switch your TV on and off, and hook into your desktop events:

https://github.com/xpusostomos/lgtv-groovy
